### PR TITLE
feat: new content map for Android docs

### PR DESCRIPTION
**Pull request by Nixon-Samuel**
depends on https://github.com/100mslive/100ms-docs/pull/1154

[content map](https://docs.google.com/spreadsheets/d/17woda3V6CoGJxgkdyc24geXB6YREVM9_s9i_5Oa-IZw/edit#gid=518237444)

- in-flight JS pages: add to content map       
   - [ ] [session metadata](https://github.com/100mslive/100ms-docs/pull/803)
- content gaps
   - [x] ordering of parent and child pages
   - [x] missing pages - call out in main spreadsheet
- impact of link changes
   - [x] redirects to old links
   - [ ] checks for 404s - especially in FAQs

### CODE REVIEW

File: .github/styles/Vocab/HMSVocab/accept.txt [hunk 1]
```diff
@@ -119,4 +119,5 @@ navbar
 upload_info
 rtmpDestinations
 hlsDestinations
-WebKit
\ No newline at end of file
+WebKit
+goswami
```
Comment (arjunattam) [inline .github/styles/Vocab/HMSVocab/accept.txt:123]: where is this used?

Comment (Nixon-Samuel) [reply .github/styles/Vocab/HMSVocab/accept.txt:123]: [changelog](https://100ms-docs-77qws43dq-100mslive.vercel.app/docs/android/v2/changelog/release-notes#200-alpha04-2021-06-04)

Comment (arjunattam) [reply .github/styles/Vocab/HMSVocab/accept.txt:123]: please remove that line. it's not useful

Comment (arjunattam) [reply .github/styles/Vocab/HMSVocab/accept.txt:123]: and this change to accept.txt

Comment (Nixon-Samuel) [reply .github/styles/Vocab/HMSVocab/accept.txt:123]: sure

**Comment by AniketSK** [comment]
@arjunattam the goswami should likely start with a capital

**Comment by arjunattam** [comment]
has the drop-down on the docs landing always been broken?
Screenshot 2023-01-24 at 4 41 30 PM

**Review by arjunattam — APPROVED** [review]

**Pull request merged**